components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "brave/components/brave_rewards/browser/rewards_service_impl.h"
 
+#include <ctime>
```

Comment:
we definitely shouldn't be using ctime in the brave code

Reply:
👍 fixed